packages/account/src/BiconomySmartAccountV2.ts

```diff
@@ -257,13 +258,105 @@ export class BiconomySmartAccountV2 extends BaseSmartContractAccount {
     return this.accountAddress;
   }
 
+  /**
+   * Transfers funds from Smart Account to recipient (usually EOA)
+   * @param recipient - Address of the recipient
+   * @param withdrawalRequests - Array of withdrawal requests {@link WithdrawalRequest}. If withdrawal request is an empty array, it will transfer the balance of the native token. Using a paymaster will ensure no dust remains in the smart account.
+   * @param buildUseropDto - Optional. {@link BuildUserOpOptions}
+   *
+   * @returns Promise<UserOpResponse> - An object containing the status of the transaction.
+   *
+   * @example
+   * import { createClient } from "viem"
+   * import { createSmartAccountClient, NATIVE_TOKEN_ALIAS } from "@biconomy/account"
+   * import { createWalletClient, http } from "viem";
+   * import { polygonMumbai } from "viem/chains";
+   *
+   * const USDT = "0xda5289fcaaf71d52a80a254da614a192b693e977";
+   * const signer = createWalletClient({
+   *   account,
+   *   chain: polygonMumbai,
+   *   transport: http(),
+   * });
+   *
+   * const smartAccount = await createSmartAccountClient({ signer, bundlerUrl, biconomyPaymasterApiKey });
+   *
+   * const { wait } = await smartAccount.withdraw(
+   *  account.pubKey, // recipient
+   *  [
+   *    { address: USDT, amount: BigInt(1) },
```

Comment:
Are they supposed to give wei value? 
or just like symbol 'USDT' and amount 0.5 is possible?

Reply:
we'd have to keep symbols and chain and token decimals mapping.
and then find out using current chainId..

Reply:
No I'm not using symbols here, it's a can of worms (we'd need a database of ticker mappings, and it'd have to be 100% reliable) as there's no easy way to match a symbol to an address. Happy to revisit this if people ask for it though. As it is here the dev will need to know their own addresses - I think that's fine.

Reply:
Yes they give gwei value. In the [tsdoc here](https://github.com/bcnmy/biconomy-client-sdk/pull/424/files#diff-77f53b7690a5e2bc101257d2fb7aab39eef4cc93964704ecb30e8b8133ec81d6R45) I've made this clear. I also think this is what devs will expect. Doing the opposite is more likely to cause confusion

Reply:
got it